unattended-upgrade

```diff
@@ -1611,6 +1608,14 @@ def try_to_upgrade(pkg,               # type: apt.Package
         pkgs_kept_back.add(pkg, cache)
 
 
+def has_candidate_pkg_version_changed(pkg,       # type: apt.Package
```

Comment:
I think `candidate_version_changed` would be a still descriptive but simpler name.

Reply:
Function renamed.